packages/assets-controllers/src/NftDetectionController.ts

```diff
@@ -155,35 +156,35 @@ export class NftDetectionController extends StaticIntervalPollingControllerV1<
 
   private getOwnerNftApi({
     address,
-    offset,
+    next,
   }: {
     address: string;
-    offset: number;
+    next?: string;
   }) {
-    return `${OPENSEA_PROXY_URL}/assets?owner=${address}&offset=${offset}&limit=50`;
+    return `${OPENSEA_PROXY_URL}/chain/ethereum/account/${address}/nfts?limit=50&next=${
```

Comment:
Not clear why the limit was 50 before.  Both V1 and V2 support up to 200.  If we're going to page through all the results anyway, why not just do 200?

Reply:
Similar comment as above — extract `ethereum`?